src/toil/utils/toilLaunchCluster.py

```diff
@@ -64,13 +67,17 @@ def main():
                              "used. For the aws provisioner this is the name of an EC2 instance "
                              "type followed by a colon and the price in dollar to bid for a spot "
                              "instance, for example 'c3.8xlarge:0.42'. Must also provide the "
-                             "--workers argument to specify how many workers of each node type "
-                             "to create.")
+                             "--workers or --managedWorkers arguments to specify how many workers of "
+                             "each node type to create.")
     parser.add_argument("-w", "--workers", dest='workers', default=None, type=str,
                         help="Comma-separated list of the number of workers of each node type to "
                              "launch alongside the leader when the cluster is created. This can be "
                              "useful if running toil without auto-scaling but with need of more "
                              "hardware support")
+    parser.add_argument("-W", "--managedWorkers", dest='managedWorkers', default=None, type=str,
```

Comment:
Hmmm... I don't know the right answer to this, but I feel like this is easily confused with `--workers`.

Some things to consider:
 - I think that the phrase "auto-scaling" should probably appear somewhere in the `--managedWorkers` help.
 - We should probably allow users to specify a minimum, something like `0-2,1-3,0-100`?

We could actually probably get away with only using `--workers` if we define that if one number is specified, then the minimum and maximum are the same and should be treated as a static number of nodes.

So the old way of:

```
--nodeTypes=node-type1,node-type2
--workers=2,2
```

would still spawn strictly two static nodes for each, but:

```
--nodeTypes=node-type1,node-type2
--workers=0-2,0-2
```

would imply and spawn autoscaling nodes with a min of 0 and a max of 2.  I think that the defined behavior is then implicit in the count definition and is more intuitive to the user.  I also think that reducing the number of options might make the interface cleaner.


It might only be more intuitive to me though.  XD  Let me know if that's the case.

Reply:
Hmm. I suppose we could make this come in through `--workers`, when you specify a range.

Reply:
OK, I think I have set it up this way now.